### ISSUE TITLE
Fix color contrast ratio of links

### ISSUE THREAD

**Issue by kaelig** [design-tokens/designtokens.org#11]
Links in the content area of the page have a color contrast ratio that doesn't meet WCAG 2.1 level AA.

![Screen Shot 2021-09-23 at 2 45 56 PM](https://user-images.githubusercontent.com/85783/134588508-ec64e91c-d13c-426e-8ef0-e84c058bfe0a.png)

**Comment by mattfelten** [comment]
Cutting it back to #a66206 on #f6f8f9 will be a contrast of 4.5. It will look more brown than gold though.

**Comment by jina** [comment]
my comments I left in Slack just now:

15:52
strange. my original colors i had checked. did they get updated?

15:53
oh, i think that’s the wrong mode link color

15:53
i have light and dark mode link colors
